meson-mode.el

```diff
@@ -110,6 +110,12 @@
       (or (eval `(or ,@meson-builtin-vars)))
       symbol-end))
 
+(defconst meson-vars-regexp
+  (rx (group (any "_" "a-z" "A-Z") (zero-or-more (any "_" "a-z" "A-Z" "0-9")))
+      (one-or-more whitespace) "=" (not (any "="))
+      )
+  )
```

Comment:
Put closing braces at the same line as last content.

Reply:
Done!